### PR TITLE
Install Agones with Helm

### LINKED ISSUE #22: Install Agones with Helm
As Open Match is installed by Helm now, let's install Agones via Helm for better deployment experience.

### PR DESCRIPTION
**Pull request by kemurayama**
This PR enables users to install Agones with Helm (Close #22)

Added following changes

- Add new `make` commands for Helm
- Trim white spaces
- Remove $ in README

### CODE REVIEW

**Comment by kemurayama** [comment]
Hi @govargo,

I made a PR for installing Agones with Helm. Could you review the PR ?

File: Makefile [hunk 1]
```diff
@@ -101,32 +110,55 @@ gcloud-test-cluster: NETWORK ?= default
...
 	kubectl delete gameservers --all --all-namespaces
-	kubectl delete -f https://raw.githubusercontent.com/googleforgames/agones/release-$(AGONES_VER)/install/yaml/install.yaml
+	helm uninstall $(AGONES_NS) --namespace $(AGONES_NS)
+	kubectl delete namespace $(AGONES_NS)
+
+# uninstall agones and agones resources in local-cluster
+.PHONY: agones-uninstall-local
+agones-uninstall-local:
+	kubectl delete fleets --all --all-namespaces
+	kubectl delete gameservers --all --all-namespaces
```
Comment (govargo) [inline Makefile:152]: Agnoes helm charat has a pre-hook to delete agones resources when chart uninstalling.
So you don't have to delete these.
https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/hooks/pre_delete_hook.yaml

Comment (kemurayama) [reply Makefile]: Thanks for reviewing. I deleted `kubectl` commands for `agones-uninstall` and `agones-uninstall-local`

Comment (govargo) [reply Makefile]: LGTM

**Review by govargo — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**